### PR TITLE
fix mangaswat

### LINKED ISSUE #6678: [Bug] <mangaswat>: <site protected open webview>
## Device information
 App version: 1.6.1 (standard, be1e7f28e, 15)
Preview build: 357
Android version: 10 (SDK 29)
Android build ID: QP1A.190711.020.A315FXXU1BUA1
Device brand: samsung
Device manufacturer: samsung
Device name: a31
Device model: SM-A315F
Device product name: a31xx
## Source information
- Name: mangaswat
- Extension version: 1.2.4

## Steps to reproduce
1. Just open the source

### Expected behavior
Manga should appear

### Actual behavior
Nothing appears

## Other details
Works fine in webview
Also tried in the preview
Some time ago in order for the site to work i had to open the webview

### PR DESCRIPTION
**Pull request by ahmedgamal17**
fix mangaswat error :"Site protected, open webview | موقع محمي ، عرض ويب مفتوح"
Closes #6678

### CODE REVIEW

File: multisrc/overrides/wpmangastream/mangaswat/src/MangaSwat.kt [hunk 1]
```diff
@@ -1,112 +1,35 @@
...
-    override fun pageListRequest(chapter: SChapter): Request {
-        return GET(baseUrl + chapter.url + "?/", headers) // Bypass "linkvertise" ads
-    }
-
-    override fun getFilterList() = FilterList(
-        StatusFilter(),
-        TypeFilter(),
-        SortByFilter(),
-        GenreListFilter(getGenrePairs())
-    )
```
Comment (CarlosEsco) [inline multisrc/overrides/wpmangastream/mangaswat/src/MangaSwat.kt:57]: is there no more searching with filters?

Comment (ahmedgamal17) [reply multisrc/overrides/wpmangastream/mangaswat/src/MangaSwat.kt:57]: Search with filters is still working but the only thing that doesn't work is filter with GENRE but the rest is working

Comment (CarlosEsco) [reply multisrc/overrides/wpmangastream/mangaswat/src/MangaSwat.kt:57]: whydid this section get removed then?  should getFilterList still need to be overidden?

Comment (ahmedgamal17) [reply multisrc/overrides/wpmangastream/mangaswat/src/MangaSwat.kt:57]: Ok I'll try and see if it work  and I'll make new PR if it works fine

Comment (ahmedgamal17) [reply multisrc/overrides/wpmangastream/mangaswat/src/MangaSwat.kt:57]: So I tested it with GENRE filters and it didn't work there is issues with it 😔😔 I'll fix it later with PR but you can merge the one first 🙂🙂

**Pull request merged**